packages/flutter/lib/src/material/slider.dart

```diff
@@ -525,6 +534,28 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
     // colors come from the ThemeData.colorScheme. These colors, along with
     // the default shapes and text styles are aligned to the Material
     // Guidelines.
+
+    final bool useV2Slider = widget.useV2Slider;
+    final double _defaultTrackHeight = useV2Slider ? 4 : 2;
+    final SliderTrackShape _defaultTrackShape = RoundedRectSliderTrackShape(useV2Slider: useV2Slider);
+    final SliderTickMarkShape _defaultTickMarkShape = RoundSliderTickMarkShape(useV2Slider: useV2Slider);
+    const SliderComponentShape _defaultOverlayShape = RoundSliderOverlayShape();
+    final SliderComponentShape _defaultThumbShape = RoundSliderThumbShape(useV2Slider: useV2Slider);
+    final SliderComponentShape _defaultValueIndicatorShape = useV2Slider ? const RectangularSliderValueIndicatorShape() : const PaddleSliderValueIndicatorShape();
+    const ShowValueIndicator _defaultShowValueIndicator = ShowValueIndicator.onlyForDiscrete;
+
+    // The value indicator color is not the same as the thumb and active track
```

Comment:
As for RangeSlider: does the developer need to understand this?

Reply:
@clocksmith , I don't think so, what do you think?

Reply:
Yes it should be mentioned in the docs